File: patterns/human-readable-names/rules-python/human_readable_names_simple.py
from openhab import rule, Registry
from openhab.actions import Transformation
from openhab.triggers import ItemStateChangeTrigger


@rule(triggers=[ItemStateChangeTrigger("MyItem")])
class MenschenlesbarerNameEinfach:
    def execute(self, module, input):
        name = Transformation.transform("MAP", "admin.map", "MyItem") or "MyItem"
        self.logger.info(name + " ist jetzt " + str(Registry.getItem("MyItem").getState()))

File: patterns/human-readable-names/rules-js/human-readable-names-simple.js
rules.JSRule({
  name: "Menschenlesbarer Name (einfach)",
  triggers: [triggers.ItemStateChangeTrigger("MyItem")],
  execute: (event) => {
    let name = actions.Transformation.transform("MAP", "admin.map", "MyItem");
    if (!name) name = "MyItem";

    console.log(name + " ist jetzt " + items.getItem("MyItem").state);
  }
});
